internal/pwru/types.go

```diff
@@ -66,6 +67,7 @@ func (f *Flags) SetFlags() {
 	flag.Uint16Var(&f.FilterDstPort, "filter-dst-port", 0, "filter destination port")
 	flag.Uint16Var(&f.FilterPort, "filter-port", 0, "filter either destination or source port")
 	flag.StringVar(&f.OutputTS, "timestamp", "none", "print timestamp per skb (\"current\", \"relative\", \"none\")")
+	flag.BoolVar(&f.OutputAbsoluteTimestamp, "output-absolute-timestamp", false, "prefix each line with the current time")
```

Comment:
Instead of creating a new param, could we extend `OutputTS` to accept `"absolute"`?

Reply:
How you want to combine the sk_buff timestamp with the absolute one?

Reply:
I'd suggest either to enable sk_buff timestamp (`current`, `relative`) or enable your user space timestamps (`absolute`) or complete disable (`none`).

Reply:
Sad to have only the choice between either sk_buff's timestamp or absolute one. I believe that we should keep the possibility to have both. 
What do you think? It's not a strong opinion but a feeling that it deserves a special flag. What are your pov?

Reply:
I'd be in favor to have as few as possible configuration options. If folks want to use both, then we can always introduce both. But for now I'd suggest to keep it in the same option.

Reply:
Know that I have the reason, I agree with you to keep as few as possible the number of config 👍🏼. 
Thx for sharing your opinion 🙇🏼.